dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove the "<classpathentry combineaccessrules..." lines, as the bnd.bnd file handles this.  Update to include:
<classpathentry kind="con" path="aQute.bnd.classpath.container"/>

Reply:
Done